slurm-on-gke/infrastructure/main.tf

```diff
@@ -0,0 +1,196 @@
+/**
```

Comment:
We have a top-level generic module for infrastructure here https://github.com/GoogleCloudPlatform/ai-on-gke/tree/main/infrastructure

Is it possible to reuse that instead of creating another infrastructure module?

Reply:
unfortunately, the infra part is tied to the described example, I can do it but an a second release. wdyt?